adapters/ddf-adapter/src/main/java/com/connexta/ion/replication/adapters/ddf/DdfNodeAdapterFactory.java

```diff
@@ -75,6 +75,8 @@ public DdfNodeAdapterFactory(
 
   @Override
   public NodeAdapter create(URL url) {
+    String certAlias = System.getProperty("javax.net.ssl.certAlias", "localhost");
```

Comment:
✏️ We may want to start co-locating all these property calls in a single class as we get more of them.

Reply:
:question: Do we really want the default to be `localhost`?

Reply:
Move the properties to ReplicationConstants. Changed the default to be `InetAddress.getLocalHost().getCanonicalHostName()` and if there is an error getting the hostname it will then default to localhost